### ISSUE TITLE
[Feedback]Request - separator for docker app and custom link

### ISSUE THREAD

**Issue by NSlas** [IceWhaleTech/CasaOS#1100]
### Description

Hi, would it be possible for you guys to create a separator or grouping for apps and external link? If it possible it would make the layout much better. Like I want to make a group folder for docker app or something, thanks

### Additional Information

	 - OS: linux
	 - CasaOS Version: 0.4.3
	 - Disk Total: 19023 
	 - Disk Used: 8808 
	 - System Info: [Debian GNU/Linux 11   \l Bit:64 Mem:11962M Core:12 Linux homeserver 5.10.0-22-amd64 #1 SMP Debian 5.10.178-3 (2023-04-22) x86_64 GNU/Linux]
	 - Remote Version: 0.4.3
	 - Browser: Edge 
	 - Version: 113.0.1774.42

**Comment by ETWang1991** [comment]
Yes, if you don't mind, could you take a screenshot and show me your dashboard. I will try to find a way to go from chaos to order
